02-analytics/technical.md

```diff
@@ -0,0 +1,377 @@
+# Technical details for the tracking plan
+
+## Implementation notes
+
+The analytics should be collected (unless otherwise noted), even when the client
+is offline. The data should be cached as far as possible and sent to the server
+when the device is back online. If caching more data becomes impossible, the oldest
+cached data should be deleted. The deletion is discussed further later in the doc.
```

Comment:
Deletion policy is missing

Reply:
Added https://github.com/bcc-code/bcc-media-adr/pull/3/files#diff-75a7681eb839d7d6bf6f8a87256ec4ce64948aa7c2207762ad61752f0e672e24R19